src/Resources/require.js

```diff
@@ -133,7 +133,10 @@
 
     window.register = function (name, initializer) {
         if (_modules[name]) {
-            throw new ModuleRedeclareError(name);
+            if (typeof console !== 'undefined' && typeof console.error === 'function') {
+                console.error(new ModuleRedeclareError(name));
```

Comment:
are we sure we want to log it as an error? Maybe a warning might be more appropriate.

Reply:
console.warn is fine to me. I would still keep it as an Error instance sent to the console as all browser will display a stack trace with the error message giving you a better dev experience on debugging.